<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderDialog</class>
 <widget class="QDialog" name="RenderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>470</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Render Settings...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="resolutionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Resolution</string>
   </property>
  </widget>
  <widget class="QComboBox" name="resolutionBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="octreeMaxDepthLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Octree Max Depth</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="octreeMaxDepthBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="octreeMinTrianglesLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Octree Min Triangles</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="octreeMinTrianglesBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="numberOfRaysLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Number of Rays</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numberOfRaysBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="sigmaLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sigma</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sigmaBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="dofLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Depth of Field</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dofOnButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dofOffButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="focalLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Focal Distance</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="focalBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RenderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RenderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
